extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1411,4 +1421,44 @@ export class AmpStoryPage extends AMP.BaseElement {
   setDescendantCssTextStyles_() {
     setTextBackgroundColor(this.element);
   }
+
+  /**
+   * Parse the description for the page.
+   * @private
+   */
+  handlePageDescription_() {
+    this.description_ = this.element.getAttribute('title');
+
+    if (this.isBotUserAgent_) {
+      this.renderPageDescription_();
+    } else {
+      // Strip the title attribute from the page on non-bot user agents, to
+      // prevent the browser tooltip.
+      this.element.removeAttribute('title');
```

Comment:
I'm not sure this is the way to go. Might be useful for accessibility. And generally we want this text to show up to incentivize it being good.

Reply:
The issue we had with this in the past, is that anchor links with no `title` will traverse the ancestors chain to find a `title` to fallback to. 
If a publisher writes a `<a href="">` with no `title`, a hover would use the page's title. Ideally all links would have titles, even if empty, but that might be harder to enforce

previous issue: https://github.com/ampproject/amphtml/issues/17654

Reply:
Is that a bug or a feature? In this case it seems the nearest slide is much better than the global per-story title.

Reply:
We also strip the global per-story title.  We can absolutely add these for accessibility, but my intention here was to make this PR in particular a no-op from a user's perspective.

Reply:
I thought the plan was also to show the string in the desktop overview?

If we want to do those things in a follow up, then I'm totally fine with merging this PR, but please describe the rationale in the comment.

Maybe we can just remove title and copy title into `aria-label` (if not present) OR use `aria-label` instead of `title`?

Reply:
Quick ping on this:
Copying to `aria-label` if not present seems like a good alternative to me.

Reply:
Yep, will do.  Have a changes locally to copy to `aria-label` for non-bots, and also properly set `aria-labelledby` on the `<div>` for bots.  Just need to clean up and push.